distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/ServiceManager.java

```diff
@@ -74,11 +74,11 @@
 
     private static final int CONFIG_UPDATE_WAIT_INTERVAL_MILLIS = 5;
 
-    public static final long MANAGED_SERVICE_TIMEOUT = TimeUnit.MINUTES.toMillis(10);
+    public static final long MANAGED_SERVICE_TIMEOUT = TimeUnit.MINUTES.toMillis(100);
```

Comment:
Do we _really_ want to use 100 and 1,000 _minutes_ here? I doubt we need or should wait that long before assuming something is wrong and giving up. Otherwise we just have a dead build consuming hardware resources for no good reason.